documentation/release_notes.rst

```diff
@@ -8,6 +8,111 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
+  ``find``, ``find_if``, ``find_end``, ``find_first_of``, ``find_if_not``, ``inclusive_scan``, ``includes``,
+  ``is_heap``, ``is_heap_until``, ``is_partitioned``, ``is_sorted``, ``is_sorted_until``, ``lexicographical_compare``,
+  ``max_element``, ``min_element``, ``minmax_element``, ``mismatch``, ``none_of``, ``partition``, ``partition_copy``,
+  ``reduce``, ``remove``, ``remove_copy``, ``remove_copy_if``, ``remove_if``, ``search``, ``search_n``,
+  ``stable_partition``, ``transform_exclusive_scan``, ``transform_inclusive_scan``, ``unique``, and ``unique_copy``
+  algorithms with device policies. 
+- Improved performance of ``sort``, ``stable_sort`` and ``sort_by_key`` algorithms with device policies when using Merge
+  sort [#fnote1]_.
+- Added ``stable_sort_by_key`` algorithm in ``namespace oneapi::dpl``. 
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``all_of``, ``any_of``,
+  ``none_of``, ``for_each``, ``find``, ``find_if``, ``find_if_not``, ``adjacent_find``, ``search``, ``search_n``,
+  ``transform``, ``sort``, ``stable_sort``, ``is_sorted``, ``merge``, ``count``, ``count_if``, ``equal``, ``copy``,
+  ``copy_if``, ``min_element``, ``max_element``. These algorithms operate with C++20 random access ranges
+  and views while also taking an execution policy similarly to other oneDPL algorithms.
+- Added support for operators ==, !=, << and >> for RNG engines and distributions.
+- Added experimental support for the Philox RNG engine in ``namespace oneapi::dpl::experimental``.
+- Added the ``<oneapi/dpl/version>`` header containing oneDPL version macros and new feature testing macros.
+
+Fixed Issues
+------------
+- Fixed unused variable and unused type warnings.
+- Fixed memory leaks when using ``sort`` and ``stable_sort`` algorithms with the oneTBB backend.
+- Fixed a build error for ``oneapi::dpl::begin`` and ``oneapi::dpl::end`` functions used with
+  the Microsoft* Visual C++ standard library and with C++20.
+- Reordered template parameters of the ``histogram`` algorithm to match its function parameter order.
+  For affected ``histogram`` calls we recommend to remove explicit specification of template parameters
+  and instead add explicit type conversions of the function arguments as necessary.
+- ``gpu::esimd::radix_sort`` and ``gpu::esimd::radix_sort_by_key`` kernel templates now throw ``std::bad_alloc``
+  if they fail to allocate global memory.
+- Fixed a potential hang occurring with ``gpu::esimd::radix_sort`` and
+  ``gpu::esimd::radix_sort_by_key`` kernel templates. 
+- Fixed documentation for ``sort_by_key`` algorithm, which used to be mistakenly described as stable, despite being
+  possibly unstable for some execution policies. If stability is required, use ``stable_sort_by_key`` instead. 
+- Fixed an error when calling ``sort`` with device execution policies on CUDA devices.
+- Allow passing iterators to C++20 random access ranges as algorithm arguments.
```

Comment:
```suggestion
- Allow passing C++20 random access iterators to oneDPL algorithms
```
It took me a while to parse this sentence because "**passing** iterators **to**" is really confusing. Only then I realized that the "iterators to C++20 random access ranges" should be read as one construction.

With my suggestion it becomes much simpler but still makes a point.